lib/view_model/active_record/update_context.rb

```diff
@@ -337,5 +341,21 @@ def try_take_released_viewmodel(vm_ref)
     def release_viewmodel(viewmodel, association_data)
       @release_pool.release_to_pool(viewmodel, association_data)
     end
+
+    # Deferred database constraints may have been violated by changes during
```

Comment:
Minor quibble: comment is describing behaviour not contained in the following code. The call to check and its motivation lives in `UpdateOperation.run!`.